spec/graphql/users/squeaks_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe 'Squeaks Query' do
+  let(:user) { create(:user) }
+  let(:squeaks) { create_list(:squeak, 10, user_id: user.id) }
+
+  it 'returns a list of squeaks' do
+    query = <<~GQL
+      query {
+        allSqueaks {
+          id
+          content
+          reports
+          nuts
+          approved
+          userId
+          createdAt
+        }
+      }
+    GQL
+
+    result = SqueakrBeSchema.execute(query, context: { all_squeaks: squeaks })
```

Comment:
I learned that we can also call the to_json method on line 22 and save having to call.dig on the result each time we want to make an assertion.